src/interfaces/StoreData.ts

```diff
@@ -0,0 +1,10 @@
+import Resource from './Resource'
+
+interface StoreData {
+    _meta: {
+        self: string
+        load: Promise<Resource>
```

Comment:
There are more known possible entries here, such as `loading?: boolean`, `deleting?: boolean` `reloading?: boolean` and `reload?: { uri: string, property: string }`

Reply:
Yep also WIP. Will add them when migrating `index.js`. The code converted so far only uses `self` and `load` and I didn't want to add properties in advance (kind of testing on the fly that TypeScript finds the issue and complains as expected).